### PR TITLE
Add icon for Reigns: Her Majesty (Steam)

### PR DESCRIPTION
**Pull request by mrmeszaros**
| Original icon | Circle icon | Square icon |
| --- | :-- | :-- |
| reigns-her-majesty original 96 | reigns-her-majesty circle 96 | reigns-her-majesty square 96 |

The game is a sequel to Reigns (#3726)

### CODE REVIEW

**Review by palob — APPROVED** [review]

**Comment by bilelmoussaoui** [comment]
The game is on Android too nop?

**Review by Foggalong — DISMISSED** [review]
[Aye it is](https://play.google.com/store/apps/details?id=com.devolver.reigns2), so the extra info needs adding

**Comment by mrmeszaros** [comment]
Could not find the `ComponentInfo` in http://activities.tundem.com.

The [prequel](https://play.google.com/store/apps/details?id=com.devolver.reigns) was also missing the android entry - that got added too.

**Review by palob — APPROVED** [review]

**Review by bilelmoussaoui — CHANGES_REQUESTED** [review]
The second android entry is not a valid one

**Comment by bilelmoussaoui** [comment]
I will make the right changes in few hours

File: data.json [hunk 1]
```diff
@@ -13531,10 +13531,21 @@
...
+        "android": [
+            "com.devolver.reigns/com.unity3d.player.UnityPlayerActivity"
+        ],
         "linux": {
             "root": "steam_icon_474750"
         }
     },
+    "reigns-her-majesty": {
+        "android": [
+            "com.devolver.reigns2"
```
Comment (mrmeszaros) [inline data.json:13543]: @bilelmoussaoui I could not find the full `ComponentInfo`

**Comment by palob** [comment]
@bilelmoussaoui Can you take a look?

**Review by Foggalong — APPROVED** [review]
At  least for now I'm approving this without the extra Android information. It'd be nice to finally get this in the Linux theme, even if the android support has to wait longer.

**Pull request merged**